vmdb/app/assets/javascripts/directives/miq_calendar.js

```diff
@@ -0,0 +1,37 @@
+miqAngularApplication.directive('miqCalendar', function() {
+  return {
+    require: 'ngModel',
+    link: function(scope, elem, attr, ctrl) {
+      console.log('miqCalendar', scope, elem, attr, ctrl);
```

Comment:
Is the console.log here really needed? Ordinarily, we don't log this sort of stuff into the console.